src/test/java/com/zendesk/maxwell/MaxwellTestWithIsolatedServer.java

```diff
@@ -38,18 +39,13 @@ public void setupSchema() throws Exception {
 	}
 
 	protected List<RowMap> getRowsForSQLTransactional(final String[] input) throws Exception {
-		MaxwellTestSupportCallback callback = new MaxwellTestSupportCallback() {
-			@Override
-			public void afterReplicatorStart(MysqlIsolatedServer mysql) throws SQLException {
-				Connection c = mysql.getNewConnection();
-				c.setAutoCommit(false);
-				for ( String s : input ) {
-					c.createStatement().execute(s);
-				}
-				c.commit();
-			}
-		};
-		return MaxwellTestSupport.getRowsWithReplicator(server, null, callback);
+		MaxwellTestSupportCallback callback = getMaxwellTestSupportCallback(input);
+		return MaxwellTestSupport.getRowsWithReplicator(server, null, callback, null);
+	}
```

Comment:
^ this one should probably just delegate to the below overload now (with `null, null`)